src/Shared/AWS/AWSSemanticConventions.v1.10_1.cs

```diff
@@ -0,0 +1,42 @@
+namespace OpenTelemetry.AWS;
+
+// disable Style Warnings to improve readability of this specific file.
+#pragma warning disable SA1124
+#pragma warning disable SA1005
+#pragma warning disable SA1514
+#pragma warning disable SA1201
+#pragma warning disable SA1516
+
+internal static partial class AWSSemanticConventions
+{
+    /// <summary>
+    /// Open Telemetry Semantic Conventions as of the 1.10.1 release of this library.
+    /// https://github.com/open-telemetry/semantic-conventions/releases/tag/v1.29.0.
+    /// </summary>
+    private class AWSSemanticConventions_v1_10_1 : AWSSemanticConventions_v1_10
+    {
+        // AWS Attributes
+        public override string AttributeAWSBedrock => "aws.bedrock";
```

Comment:
Is the pattern forcing us to create a new SemanticConventions every time we add a new service? Seems like if we are being additive that shouldn't require us to define a new semantic conventions version.